apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -81,6 +88,28 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     } as const;
   }
 
+  let teamMemberEmail = null;
+
+  if (eventData.schedulingType === SchedulingType.ROUND_ROBIN && email) {
```

Comment:
Move finding the CRM contact owner from `useSchedule()` to `getServerSideProps()`